tools/azsdk-cli/Azure.Sdk.Tools.Cli/Configuration/RequirementsV1.json

```diff
@@ -0,0 +1,121 @@
+{
+    "categories": {
+        "core": [
```

Comment:
the verify readme common script uses python and pip - https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/scripts/Verify-Readme.ps1#L85, it may be worth considering adding it to the core requirements since I think that script will be run for all languages in the validate step (might be wrong tho)